src/main/java/com/redhat/devtools/lsp4ij/LanguageServerFactory.java

```diff
@@ -50,4 +51,7 @@ public interface LanguageServerFactory {
         return LanguageServer.class;
     }
 
+    @NotNull default LSPCustomizationServerSupport createServerSupport(@NotNull Project project) {

```

Comment:
>Shouldn't the adjective form be used here (i.e., LSPCustomizedServerSupport)? Same for other places.

Thanks fixed! (My english is not perfect)

Reply:
> For comparison, the native client uses LspDiagnosticsSupport (with lowercase sp).

I have seen that, but our existing code uses `LSP`, so I would like to avoid doing this refactoring.

Reply:
After discussing with @fbricon we decide to call that LSPClientFeatures, hope you will like it.